deploy/template.yaml

```diff
@@ -2564,6 +2564,76 @@ Resources:
       FilterPattern: ""
       LogGroupName: !Ref BulkMigrateVcsFunctionLogGroup
 
+  ReconcileMigratedVcsFunction:
```

Comment:
Will this lambda automatically incorporate the DynaTrace layer,  if so is the memory allocation sufficient?

Reply:
Yeah it will - and the memory as configured should be fine - it's how all our other lambdas run, including the bulk migration one.
Having said that, I've been playing around with memory size and how that affects the performance/cost of the lambda. And I'm going to add a commit to max it out. The speed up is significat, and the extra cost is approx 1 cent per invocation. Seems worth it.